src/components/UserInfo.js

```diff
@@ -1,16 +1,22 @@
 class UserInfo {
-  constructor(nameSelector, jobSelector) {
+  constructor(nameSelector, aboutSelector, avatarSelector) {
     this._name = document.querySelector(nameSelector);
-    this._job = document.querySelector(jobSelector);
+    this._about = document.querySelector(aboutSelector);
+    this._avatar = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
-    return { name: this._name.textContent, job: this._job.textContent };
+    return { name: this._name.textContent, about: this._about.textContent };
   }
 
-  setUserInfo({ name, job }) {
-    this._name.textContent = name;
-    this._job.textContent = job;
+  setUserAvatar(link) {
+    this._avatar.src = link;
+  }
+
+  setUserInfo({ name, about, avatar }) {
+    if (name) this._name.textContent = name;
+    if (about) this._about.textContent = about;
+    if (avatar) this._avatar.src = avatar;
```

Comment:
`setUserAvatar` duplicates the code in line 19. Please, use it here or remove `setUserAvatar`